plugins/treasuredata/test-fixtures/td.conf

```diff
@@ -0,0 +1,3 @@
+[account]
+  user = user@example.com
+  apikey = 1/xxx
```

Comment:
Final nit: it would be nice if the test fixture would use a more realistically looking value, like we do in [other plugins](https://github.com/1Password/shell-plugins/blob/8b715d0cd6f469f5142ccfad9df49ead3c149945/plugins/aws/test-fixtures/credentials). You can generate such a value using the `make treasuredata/example-secrets` command.